index.html

```diff
@@ -272,134 +315,188 @@ <h2>Glossary</h2>
+<p><dfn class="lint-ignore export">Language priority list</dfn>. A collection of one or more [=language ranges=] identifying the user's language preferences for use in matching. As the name suggests, such lists are normally ordered or weighted according to the user's preferences. The HTTP [[RFC2616]] <code>Accept-Language</code> [[RFC3282]] header is an example of one kind of language priority list.</p>
 
-<p><dfn data-lt="language range|range|language-range|language ranges|Language range" id="def_language_range" class="lint-ignore export">Language range</dfn>. A string similar in structure to a language tag that is used for "identifying sets of language tags that share specific attributes".</p>
+<p><dfn data-lt="language-range" class="lint-ignore export">Language range</dfn>. A string similar in structure to a language tag that is used for "identifying sets of language tags that share specific attributes".</p>
 
-<p><dfn data-lt="subtag|subtags|Subtag" id="def_subtag" class="lint-ignore export">Language subtag</dfn>. A sequence of ASCII letters or digits separated from other subtags by the hyphen-minus character and identifying a specific element of meaning withing the overall <a href="#def_language_tag">language tag</a>. In [[BCP47]], subtags can consist of upper or lowercase ASCII letters (the case carries no distinction) or ASCII digits. Subtags are limited to no more than eight characters (although additional length restrictions apply depending on the specific use of the subtag).
+<p><dfn class="lint-ignore export">Language subtag</dfn>. A sequence of ASCII letters or digits separated from other subtags by the hyphen-minus character and identifying a specific element of meaning withing the overall [=language tag=]. In [[BCP47]], subtags can consist of upper or lowercase ASCII letters (the case carries no distinction) or ASCII digits. Subtags are limited to no more than eight characters (although additional length restrictions apply depending on the specific use of the subtag).
 </p>
-<p><dfn data-lt="language tag|language tags|Language tag" id="def_language_tag" class="lint-ignore export">Language tag</dfn>. A string used as an identifier for a language, usually referring explicitly to a [[BCP47]] language tag. These language tags consist of one or more subtags.
+<p><dfn class="lint-ignore export">Language tag</dfn>. A string used as an identifier for a language, usually referring explicitly to a [[BCP47]] language tag. These language tags consist of one or more subtags.
```

Comment:
link `subtags` to the term `language subtag` just above?

```suggestion
<p><dfn class="lint-ignore export">Language tag</dfn>. A string used as an identifier for a language, usually referring explicitly to a [[BCP47]] language tag. These language tags consist of one or more [=language subtags=].
```